doc/guide/CircleScrollView.md

```diff
@@ -5,25 +5,24 @@ summary: CircleScrollView control guide
 
 |![Horizontal](data/CircleScrollView_Horizontal.png)|![Vertical](data/CircleScrollView_Vertical.png)|
 |:-----------------------------------------------:|:-----------------------------------------------:|
 |                      Horizontal                 |                    Vertical                     |
 
-**WARNING: [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) must be confined in the `CirclePage` container or [Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/) with [CircleSurfaceEffectBehavior](xref:Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior). If you add these controls in any other way,  it may cause an exception or cannot display the controls.**
+## Create CircleScrollView
 
-## Add CircleScrollView in CirclePage
+Basically `CircleScrollView` looks same as [Xamarin.Forms.ScrollView](https://developer.xamarin.com/api/type/Xamarin.Forms.ScrollView/), scrolling is also possible with [Bezel interaction](https://developer.tizen.org/design/wearable/interaction/bezel-interactions).
+The difference from [Xamarin.Forms.ScrollView](https://developer.xamarin.com/api/type/Xamarin.Forms.ScrollView/) is to provide some additional property for Tizen wearable such as `BarColor`
 
-You can set `CircleScrollView` in [CirclePage.Content](xref:Tizen.Wearable.CircularUI.doc.CirclePage). For more information about how to add a [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage), see [CirclePage guide](https://samsung.github.io/Tizen.CircularUI/guide/CirclePage.html#create-circlepage).
-The following XAML code shows [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage) with `CircleScrollView`.
-`RotaryFocusObject` property sets the currently focused control that is handled by rotating and displays the circle object of focused control.
-If the value is not set properly, the control does not receive the [Rotary Event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary).
+`CircleScrollView` has the following property:
 
-The direction of the scroller depends on the setting of the `Orientation` value.
+- BarColor: This property gets or sets a scroll bar color value.
 
-In the following example, the `Orientation` of the `CircleScrollView` is set to `Horizontal` and is placed in the `StackLayout` to contain many images:
+The following XAML code shows how to use `CircleScrollView`.
+The direction of the scroller depends on the setting of the `Orientation` value.
+In the following example, the `Orientation` of the `CircleScrollView` is set to `Vertical` and is placed in the `StackLayout` to contain many images and `BarColor` is "Red"
```

Comment:
In the following example, the `Orientation` of the `CircleScrollView` is set to `Vertical`. `CircleScrollView` is placed in the `StackLayout` to contain many images, and its `BarColor` is "Red".